MANIFEST.in

```diff
@@ -16,6 +16,7 @@ exclude CODE_OF_CONDUCT.md
 exclude docs
 exclude mypi.ini
 exclude readthedocs.yml
+exclude requirements.in
```

Comment:
This means we won't package up `requirements.in` in the package, correct? I think that will be problematic.